bundles/io/org.eclipse.smarthome.io.rest.core/src/main/java/org/eclipse/smarthome/io/rest/core/discovery/InboxResource.java

```diff
@@ -108,6 +108,13 @@ public Response ignore(@PathParam("thingUID") String thingUID) {
         inbox.setFlag(new ThingUID(thingUID), DiscoveryResultFlag.IGNORED);
         return Response.ok().build();
     }
+    
+    @POST
+    @Path("/unignore/{thingUID}")
```

Comment:
shouldn't the endpoint better be inbox/<id>/unignore in a good REST manner (the ignore one of course as well then)?